spec/models/form/sales/questions/staircase_spec.rb

```diff
@@ -7,6 +7,10 @@
   let(:question_definition) { nil }
   let(:page) { instance_double(Form::Page, subsection: instance_double(Form::Subsection, form: instance_double(Form, start_date: Time.zone.local(2023, 4, 1)))) }
 
+  before do
+    allow(page.subsection.form).to receive(:start_year_2025_or_later?).and_return(false)
+  end
```

Comment:
Given there's now logic on this question around the answer options, I think we should test that the answer options are correct for "before 2025" and "2025 onwards".